tools/scripts/diff-sizes.mjs

```diff
@@ -82,6 +89,7 @@ async function getBuildSize (buildType) {
   }
 
   const buildStats = await fs.readJson(buildStatsPath)
+  if (buildType === 'npm') console.log(buildStatsPath)
```

Comment:
```suggestion
```

Reply:
I'm thinking this was a leftover from testing stuff. If not, curious why it's logging only for 'npm' build type.

Reply:
TY